app/events/[event_id]/page.tsx

```diff
@@ -0,0 +1,277 @@
+import { fetchEventByID, fetchEventHost } from '@/api/supabase/queries/events';
+import {
+  fetchFacilityByID,
+  fetchFacilityContactByID,
+} from '@/api/supabase/queries/facility';
+import { fetchPerformer } from '@/api/supabase/queries/volunteers';
+import LocPin from '@/public/images/black_loc_pin.svg';
+import BPLogo from '@/public/images/bp-logo.png';
+import Calendar from '@/public/images/calendar_icon.svg';
+import FacilityContactPin from '@/public/images/facility_contact_icon.svg';
+import HostPin from '@/public/images/host_icon.svg';
+import ProducerIcon from '@/public/images/producer_icon.svg';
+import PerformerPin from '@/public/images/volunteer_performer_icon.svg';
+import WhiteBack from '@/public/images/white_back.svg';
+import COLORS from '@/styles/colors';
+import {
+  Event,
+  Facilities,
+  FacilityContacts,
+  Volunteers,
+} from '@/types/schema';
+import formatTime from '@/utils/formatTime';
+import * as styles from './styles';
+
+export default function EventDisplay({
+  params,
+}: {
+  params: { event_id: UUID };
+}) {
+  const [event, setEvent] = useState<Event>();
+  const [facility, setFacility] = useState<Facilities>();
+  const [facility_contact, setFacilityContact] = useState<FacilityContacts>();
+  const [host_name, setHostName] = useState<string>();
+  const [host_phone_number, setHostPhoneNumber] = useState<string>();
+  const [performer, setPerformer] = useState<Volunteers>();
+
+  useEffect(() => {
+    const getEvent = async () => {
+      const fetchedEvent: Event = await fetchEventByID(params.event_id);
+      setEvent(fetchedEvent);
+      const fetchedFacility: Facilities = await fetchFacilityByID(
+        fetchedEvent.facility_id,
+      );
+      setFacility(fetchedFacility);
+      const fetchedFacilityContact: FacilityContacts =
+        await fetchFacilityContactByID(fetchedEvent.facility_id);
+      setFacilityContact(fetchedFacilityContact);
+
+      if (fetchedEvent.needs_host) {
+        const host: Volunteers = await fetchEventHost(params.event_id);
+        setHostName(`${host.first_name} ${host.last_name}`);
+        setHostPhoneNumber(host.phone_number);
+      } else {
+        setHostName(fetchedFacility.host_name);
+        setHostPhoneNumber(fetchedFacility.host_contact);
+      }
+
+      const fetchedPerformer: Volunteers = await fetchPerformer(
+        params.event_id,
+      );
+      setPerformer(fetchedPerformer);
+    };
+    getEvent();
+  }, [params.event_id]);
+
+  // Render once the event is fetched
+  if (!event || !facility || !facility_contact || !performer) {
+    return <p></p>;
+  }
+
+  const time = formatTime(
+    new Date(event.start_date_time),
+    new Date(event.end_date_time),
+    true,
+  );
+  const location = facility.street_address_1 + ', ' + facility.city;
+
+  return (
+    <styles.Page>
+      <styles.ImageWrapper>
+        <styles.EventImage src={BPLogo} alt="EventImage" />
+        <styles.GradientOverlay />
+      </styles.ImageWrapper>
```

Comment:
delete these are replace with this:
`<styles.EventImage src={BPLogo} alt="EventImage" />`